### ISSUE TITLE
Disable "reports not yet processed" message

### ISSUE THREAD

**Issue by ghost** [piwik/piwik#19539]
## Summary

I'd like to be able to turn off the blue boxed message at the top of the dashboard that says "It looks like reports for this period have not been processed yet. Do you want to see what's happening now? Check out the Visits log or choose a different date period until the reports are generated."  This message is annoying and does not convey useful information in my installation.

In more detail:  my reports are processed by a cron job and they are always as up-to-date as I need them to be.  I have several sites in a single Matomo installation, some of which get very few hits (averaging about one hit per day) and the "reports not yet processed" message comes up whenever there has been no hit on a given site yet in the current day - which is the whole day if there is no hit that day, and on average half the day on a day with only one hit, those cases together being most days.  So the message is reminding me near-daily, and requiring a click to acknowledge near-daily, of information that I already know (because I know how I set up my archiving and have seen the message before), that isn't really applicable to my case (because my existing cron job will process the reports promptly when hits do come in), and that duplicates what is already shown on the same screen anyway (because the mentioned "Visits log" is right there on the dashboard below the blue-boxed message).

I'd like to have at least one of:
 - a "dismiss permanently" button in the message box instead of or in addition to the existing "close this time" X icon
 - a configuration option somewhere to prevent this message from coming up
 - this message removed from the software entirely

## Your Environment

* Matomo Version: 4.10.1
* PHP Version: 7.4.20
* Server Operating System: Linux
* Additionally installed plugins:

**Comment by sgiehl** [comment]
Hi @mskala 
Thanks for your suggestion.
So this notification is shown when there had not yet been any visit to that specific site today? Iirc the message should only be shown when there had been a visit, but the data had not yet been processed. Maybe there is a bug around that logic. I'll have a quick look.

**Comment by sgiehl** [comment]
I found a bug in this handling. See #19547. But as I didn't fully try to reproduce your issue, I'm unable to say if that will fix it.

**Comment by ghost** [comment]
Yes, in my current installation it appears to be shown when the dashboard is set to "today" and there have been no visits today so far, even though reports have actually been processed.  Sounds like that isn't the intended behaviour.

Ideally, I would prefer to be able to disable this message even for the apparently intended cases, i.e. the few minutes between midnight and the first run of the report-processing cron job.
